samples/client/petstore/perl/t/AnotherFakeApiTest.t

```diff
@@ -0,0 +1,41 @@
+=begin comment
```

Comment:
I don't think this file should be included ...

We've cleared this issue (corrupted file?) in the latest master.

Reply:
This file has been removed